web/router.ex

```diff
@@ -17,6 +18,8 @@ defmodule Notox.Router do
     pipe_through :browser # Use the default browser stack
 
     get "/", PageController, :index
+    resources "/users", UserController
+    resources "/session", SessionController, only: [:new, :create, :delete]
```

Comment:
This is how the book did it, so I followed along. But shouldn't this be a singleton? I don't see the point of having a session ID, I just need to be able to create and delete sessions.